src/python/pants/option/global_options.py

```diff
@@ -229,6 +229,7 @@ def register_bootstrap_options(cls, register):
     register('--backend-packages', advanced=True, type=list,
              default=['pants.backend.graph_info',
                       'pants.backend.python',
+                      'pants.backend.python.lint.isort',
```

Comment:
I hate registering V1 isort by default, but I can't think of a deprecation strategy to work cleanly. I also don't like deprecating V1 isort being enabled by default at the same time that we are deprecating `--fmt-isort-skip` (removed in 1.27).